src/main/java/org/broadinstitute/dsde/consent/ontology/translate/DTO/RecommendationDTO.java

```diff
@@ -0,0 +1,24 @@
+package org.broadinstitute.dsde.consent.ontology.translate.DTO;
+
+public class RecommendationDTO {
```

Comment:
The java naming conventions most DSP projects follow (based on the google java conventions) require that package names are all lower case and that initialisms like `DTO` only have the first letter capitalized. 
```suggestion
package org.broadinstitute.dsde.consent.ontology.translate.dto;

public class RecommendationDto {
```